solutions/numpy_core_tests/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM python:3.9-slim
 
-RUN pip3 install numpy pytest hypothesis &&\
+RUN pip3 install numpy==1.22.1 pytest hypothesis &&\
```

Comment:
There was the argument of keeping numpy at the latest version. This way we can know of emerging incompatibilities with numpy head.
Of course this comes with the cost of fixing the issues everytime a numpy dependency changes.
cc: @jxyang

Reply:
cc @paulcallen

Reply:
Do we know what is broken with numpy 1.22.2? I am fine with fixing the pipeline this way, provided we enable it back for 1.22.2 soon.

Reply:
``` 
*** kernel panic: syscall.c(7587): _syscall(): unhandled syscall: SYS_listxattr()
[2022-02-04T23:03:38.763Z] 0x1007f8b30: __myst_panic()
[2022-02-04T23:03:38.763Z] 0x1007d779a: _syscall()
[2022-02-04T23:03:38.763Z] 0x10081e0b5: __morestack()
[2022-02-04T23:03:38.763Z] Fatal Python error: Segmentation fault
[2022-02-04T23:03:38.763Z] 
[2022-02-04T23:03:38.763Z] Current thread 0x00000001c1ee7828 (most recent call first):
[2022-02-04T23:03:38.763Z]   File "/usr/local/lib/python3.9/shutil.py", line 330 in _copyxattr
[2022-02-04T23:03:38.763Z]   File "/usr/local/lib/python3.9/shutil.py", line 388 in copystat
[2022-02-04T23:03:38.763Z]   File "/usr/local/lib/python3.9/shutil.py", line 445 in copy2
[2022-02-04T23:03:38.763Z]   File "/usr/local/lib/python3.9/shutil.py", line 508 in _copytree
[2022-02-04T23:03:38.763Z]   File "/usr/local/lib/python3.9/shutil.py", line 566 in copytree
[2022-02-04T23:03:38.763Z]   File "/usr/local/lib/python3.9/site-packages/numpy/core/tests/test_limited_api.py", line 25 in test_limited_api
```
This test "test_limited_api" is a new test file, introduced in numpy 1.22.2 (released Feb 02), 
https://github.com/numpy/numpy/releases/tag/v1.22.2 in this PR: 
https://github.com/numpy/numpy/pull/20818/files

Related line of code https://github.com/numpy/numpy/blob/main/numpy/core/tests/test_limited_api.py#L25

Reply:
`test_limited_api` requires support of SYS_listxattr with numpy head.